docs/resources/connector_azure_vnet.md

```diff
@@ -10,49 +10,54 @@ description: |-
 
 Manage Azure Cloud Connector.
 
+This connector requires at least two other resources `alkira_segment`
+and `alkira_credential_azure_vnet` to work. Optionally, it works with
+`alkira_group`, `alkira_billing_tags` and several other resources.
+
+
 ## Example Usage
 
-```terraform
-#
-# Create segments assuming this is the completely empty tenant network
-#
-resource "alkira_segment" "segment1" {
-  name = "seg1"
-  asn  = "65513"
-  cidr = "10.16.1.0/24"
-}
+The following examples assumes that `alkira_segment` and
+`alkira_credential_azure_vnet` are already created.
 
+A simple connector could be created like this:
 
-#
-# Create the credential to store the access to the AWS account that
-# VPCs belongs two. In this example, both VPCs belong to this AWS
-# account.
-#
-resource "alkira_credential_azure_vnet" "customer_azure" {
-  name            = "customer-azure"
-  application_id  = ""
-  secret_key      = ""
-  subscription_id = ""
-  tenant_id       = ""
+```terraform
+resource "alkira_connector_azure_vnet" "connector1" {
+  name           = "test-vnet1"
+  azure_region   = "westus2"
+  azure_vnet_id  = "/subscriptions/XXXX/resourceGroups/Test/providers/Microsoft.Network/virtualNetworks/test-vnet1"
+  credential_id  = alkira_credential_azure_vnet.yours.id
+  cxp            = "US-WEST"
+  group          = "test"
+  segment        = alkira_segment.segment1.name
+  size           = "SMALL"
 }
+```
 
+Additionally, you could specify the routing options by using the
+`routing` block along with resource `alkira_policy_prefix_list`:
 
-#
-# Create AZURE-VNET connector for the first VNET and attach it with
-# segment 1
-#
-resource "alkira_connector_azure_vnet" "connector_vnet1" {
-  name           = "customer-vnet1"
+```terraform
+resource "alkira_connector_azure_vnet" "connector2" {
+  name           = "test-vnet2"
   azure_region   = "westus2"
-  azure_vnet_id  = "/subscriptions/XXXX/resourceGroups/Test/providers/Microsoft.Network/virtualNetworks/customer-vnet1"
-  credential_id  = alkira_credential_azure_vnet.customer_azure.id
+  azure_vnet_id  = "/subscriptions/XXXX/resourceGroups/Test/providers/Microsoft.Network/virtualNetworks/test-vnet2"
+  credential_id  = alkira_credential_azure_vnet.yours.id
   cxp            = "US-WEST"
   group          = "test"
   segment        = alkira_segment.segment1.name
   size           = "SMALL"
+
+  routing {
+    options         = "AVERTISE_CUSTOM_PREFIX"
+    prefix_list_ids = [alkira_policy_prefix_list.test.id]
```

Comment:
options as a typo and a missing closing bracket
```suggestion
  routing {
    options         = "ADVERTISE_CUSTOM_PREFIX"
    prefix_list_ids = [alkira_policy_prefix_list.test.id]
  }
```

Reply:
Yep, let me rework to get rid of this block